test/run-k8s-integration.sh

```diff
@@ -23,6 +23,7 @@ export GCE_PD_VERBOSITY=9
 make -C ${PKGDIR} test-k8s-integration
 
 base_cmd="${PKGDIR}/bin/k8s-integration-test \
+            --kube-feature-gates=\"ExpandCSIVolumes=true\"
```

Comment:
can you add this to the local script too (or a comment saying to bring up your cluster with this feature gate or it wont work if the script doesn't bring up a cluster)

Reply:
I'm actually about to remove this, since we don't want resize test in latest yet